code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -105,11 +105,14 @@
 	icon_state = "psyamp"
 	item_state = "psyamp"
 
-/obj/item/clothing/suit/wizrobe/gentlecoat
+/obj/item/clothing/suit/storage/toggle/wizrobe/gentlecoat

```

Comment:
This path change is meaningful: there's stuff defined at `/obj/item/clothing/suit/wizrobe` I believe.

Reply:
Yes, It is need to have same traits as `/obj/item/clothing/suit/wizrobe`, change it.